cli/args/flags.rs

```diff
@@ -1734,6 +1735,7 @@ fn compile_args(app: Command) -> Command {
   app
     .arg(import_map_arg())
     .arg(no_remote_arg())
+    .arg(no_npm_arg())
```

Comment:
I don’t really understand the value of this flag. Once we have support for deno info, people will be able  to easily check if any imports have resolved to an npm package. I don’t think we should add this.